corehq/apps/hqcase/views.py

```diff
@@ -89,10 +89,13 @@ def delete_cases(self, request, domain):
 @requires_privilege_with_fallback(privileges.API_ACCESS)
 @api_throttle
 def case_api(request, domain, case_id=None):
+    # TODO: Make all handle functions location-safe,
```

Comment:
Is this for a separate PR?

Reply:
Yes.

Reply:
([SC-2650](https://dimagi-dev.atlassian.net/browse/SC-2650))